soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -1173,4 +1173,21 @@ struct itim64_reg {
 #define NPCX_ITCTSXX_CKSEL               4
 #define NPCX_ITCTSXX_ITEN                7
 
+/* Debug Interface registers */
+struct dbg_reg {
+	/* 0x000: Debug Control */
```

Comment:
Is there another commit/pull request with the device tree entry?  I see the base address in the Nuvoton documentation lists the base address 0x400C3000, and the DBGCTRL at offset 0x74.  While this sets to the offset to 0.

Reply:
The device tree node for dbg is under platfome/ec/zephyr because there seems no appropriate driver/dts folder to put dts/yaml files for dbg node.
Please refer to the CL: [here](https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/2712142)
The base address 0x400c3000 of dbg is the same as that of scfg in datasheet. So I set the base address of dbg DT entry to 0x400c3074.
That's why the register offset starts from 0x0.